src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -43,6 +119,7 @@ public void isDueOn_monthlySimple() {
         LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
         assertTrue(policy.isDueOn(oneSaturday));
     }
+
     @Test
     public void isDueOn_monthlyFiltered() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)